doc/nrf/libraries/others/event_manager_proxy.rst

```diff
@@ -55,9 +55,9 @@ To implement an Event Manager Proxy, you must complete the following steps:
     .. code-block:: c
 
        ret = event_manager_proxy_add_remote(ipc1_instance);
-       /* Error handling */
+       /* Error handling (decide if -EALREADY is treated as an error) */
```

Comment:
I don't think we need to explicitly add this here. I would remove those lines. `/* Error handling */` is sufficient

Reply:
Ok